projects/cdk/schematics/ng-update/constants/consts.ts

```diff
@@ -122,4 +122,44 @@ export const CONSTS_TO_REPLACE: ReplacementConst[] = [
             moduleSpecifier: '@taiga-ui/kit',
         },
     },
+    {
+        from: {
+            name: 'TuiPdfViewerModule',
+            moduleSpecifier: '@taiga-ui/proprietary-banking',
+        },
+        to: {
+            name: 'TuiPdfViewerModule',
+            moduleSpecifier: '@taiga-ui/kit',
+        },
+    },
+    {
+        from: {
+            name: 'TuiPreviewHostModule',
+            moduleSpecifier: '@taiga-ui/proprietary-banking',
+        },
+        to: {
+            name: 'TuiPreviewModule',
+            moduleSpecifier: '@taiga-ui/addon-preview',
+        },
+    },
+    {
+        from: {
+            name: 'TuiCarouselModule',
```

Comment:
Proprietary Carousel is old and has different API. We should show a warning to check the new version docs.

Reply:
fixed